kube/services/jobs/config-fence-job.yaml

```diff
@@ -0,0 +1,94 @@
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: config-fence
+spec:
+  # not yet supported - backOffLimit: 3
+  template:
+    metadata:
+      labels:
+        app: gen3job
+    spec:
+      serviceAccountName: useryaml-job
+      volumes:
+        - name: shared-data
+          emptyDir: {}
+        - name: creds-volume
+          secret:
+            secretName: "fence-creds"
+        - name: config-helper
+          configMap:
+            name: config-helper
+      containers:
+      - name: fence
+        image: quay.io/cdis/fence:master
+        imagePullPolicy: Always
+        env:
+          - name: PYTHONPATH
+            value: /var/www/fence
+        volumeMounts:
+          - name: "creds-volume"
+            readOnly: true
+            mountPath: "/var/www/fence/creds.json"
+            subPath: creds.json
+          - name: "config-helper"
+            readOnly: true
+            mountPath: "/var/www/fence/config_helper.py"
+            subPath: config_helper.py
+          - name: shared-data
+            mountPath: /mnt/shared
+        command: ["/bin/bash"]
+        args:
+          - "-c"
+          # Script always succeeds if it runs (echo exits with 0)
+          - |
+            # install json query helper if necessary ...
+            if ! which jq > /dev/null; then
+              export DEBIAN_FRONTEND=noninteractive
+              sudo -E apt install -y jq
+            fi
```

Comment:
It doesn't look like you need `jq` - remove this block?

Reply:
I don't thing jq is in the fence docker, is it?